website/docs/compile.md

```diff
@@ -70,6 +70,13 @@ scala-cli compile Hello.scala -O -deprecation -O -Xlint:infer-any
 
 `-O` accepts both options with the prefixes above and those without such a prefix.
 
+## Scala compiler plugins
+Use `--compiler-plugin` to add compiler plugin dependencies:
```

Comment:
Seems `--compiler-plugin` was removed. (We should keep it IMO, along with `-P` and `--plugin`.)